WordPress/Classes/ViewRelated/Comments/CommentsTableViewCell.swift

```diff
@@ -6,22 +6,25 @@ open class CommentsTableViewCell: WPTableViewCell {
     // MARK: - IBOutlets
 
     @IBOutlet private weak var gravatarImageView: CircularImageView!
-    @IBOutlet private weak var detailsLabel: UILabel!
+    @IBOutlet private weak var titleLabel: UILabel!
+    @IBOutlet private weak var detailLabel: UILabel!
     @IBOutlet private weak var timestampImageView: UIImageView!
     @IBOutlet private weak var timestampLabel: UILabel!
 
     // MARK: - Private Properties
 
-    private var author: String?
-    private var postTitle: String?
-    private var content: String?
+    private var author = String()
+    private var postTitle = String()
+    private var content = String()
```

Comment:
I don't know why, but I was amused by the use of constructors rather than empty literals 😄